editor.planx.uk/src/ui/public/ExternalPlanningSiteDialog.tsx

```diff
@@ -53,7 +53,7 @@ const getTitleAndContent = (
 
 interface Props {
   purpose: DialogPurpose;
-  teamSettings?: TeamSettings;
+  teamSettings?: Partial<TeamSettings>;
```

Comment:
nit: It actually looks like this prop is never used (please check this!) - this means we should remove this 👍 The settings are fetched via the store rather than being passed in as props.